engine/include/engine/engine.h

```diff
@@ -8,12 +8,34 @@
 struct ResultSet {
 	std::vector<gdf_column *> columns;
 	std::vector<std::string> names;
-	ral::frame::BlazingTableView *blazingTableView;
+	std::unique_ptr<ral::frame::BlazingTable> blazingTable;
+
+	ResultSet() {
+		columns = {};
+		names = {};
+		blazingTable = nullptr;
+	}
+
+	ResultSet(std::unique_ptr<ral::frame::BlazingTable> blazingTableValue)
```

Comment:
in other parts of the code we are writing this like so 

```
BlazingTable::BlazingTable(
  std::unique_ptr<CudfTable> table,
  std::vector<std::string> columnNames)
  : table(std::move(table)), columnNames(columnNames){

}
```
 so change the forward to just use the move. this allows you to also make declaration simpler.